### ISSUE TITLE
Temporal Stimulation Bionic causing errors

### ISSUE THREAD

**Issue by Alexalmighty502** [chaosvolt/cdda-arcana-mod#169]
(please bear with me I'm completely new with github)
CDDA Version: Build #11169
Arcania version: (5c51004 last updated 6 hours ago from the time of writing)

Issue: When enabling Temporal Stimulation Bionic the following error occurs 
 DEBUG    : invalid trait id "ARCANA_TEMPORAL_STIMULATION"

 FUNCTION : const T& generic_factory<T>::obj(const string_id<T>&) const [with T = mutation_branch]
 FILE     : src/generic_factory.h
 LINE     : 433

It still drains my battery however I don't seem to notice any difference between it being on and off other than that.

**Comment by chaosvolt** [comment]
Oh wh-okay, I think I see where I dun goofed. Holy shit, thank you for finding that, I'm surprised that didn't error on load.

It's a leftover of an earlier WIP where I tested trying to apply the effects of the bionic via a mutation-from-enchantment, but when that didn't work I ended up reworking it to rely solely off standard enchantment values. I must've deleted the mutation it needed but not the reference to it in the enchantment.
